interface/main/calendar/add_edit_event.php

```diff
@@ -1428,6 +1545,33 @@ function find_available(extra) {
   </td>
  </tr>
 
+    <style>
+        #days_every_week_row input[type="checkbox"]{float:right;}
+        #days_every_week_row div{display: inline-block; text-align: center; width: 12%;}
+        #days_every_week_row div input{width: 100%;}
+    </style>
+
+<tr id="days_every_week_row">
+    <td></td>
+    <td></td>
+    <td><input  type='checkbox' name='days_every_week' onclick='set_days_every_week()' <?php if (isDaysEveryWeek($repeats)) echo " checked" ?>/></td>
+    <td id="days_label"><?php echo xlt('Days Of Week:'); ?></td>
+    <td id="days">
+        <?php
+        foreach (array(1 => xl('Su') , 2 => xl('M'), 3 => xl('Tu'), 4 => xl('W'),
+                     5 => xl('Th'), 6 => xl('F'), 7 => xl('Sa')) as $key => $value)
```

Comment:
Note these are basically a comment to the translator and these stuff within the brackets are not shown when displayed.